applications/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -19,8 +19,8 @@
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_BLE_SCANNING_LOG_LEVEL);
 
 
-#define DEVICE_NAME	CONFIG_DESKTOP_BLE_SHORT_NAME
-#define DEVICE_NAME_LEN	(sizeof(DEVICE_NAME) - 1)
+#define DEVICE_NAME_MOUSE	"Mouse nRF52"
```

Comment:
We will have to extend the config somehow to allow more strings to filter.

Reply:
Filter added as separate header file - discussed offline.